comments-by-guideline-and-success-criterion.md

```diff
@@ -566,15 +566,17 @@ See also the [Comments on Closed Functionality](#comments-on-closed-functionalit
 
 ###### Applying SC 2.4.2 Page Titled to Non-Web Documents and Software
 
-This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.4.2](https://www.w3.org/WAI/WCAG22/Understanding/page-titled#intent) replacing “Web pages” with “non-web documents or software”.
+This should **not** be applied directly as written to non-web documents or software.  Titles for non-web software or documents must provide descriptive identification but titles almost never describe topic or purpose.
```

Comment:
I think this goes too far by removing documents from meeting the requirement. Documents typically will have a title inside of the file. I don't think this requirement is really talking about the file name which often doesn't have any meaningful information and may simply be a date or the like.

Reply:
I am all for splitting non-web documents from non-web software.  I will make edits after the call today.

Reply:
@bruce-usab Thanks! I think we've got non-web document content agreed at this point, so it would be nice to have the PR reflect a split and the agreed content for docs. in there. See the top part of Proposal 3 in the Google doc for a clean version of the text.